docs/get-started/getting-started-gcp.md

```diff
@@ -7,7 +7,7 @@ parent: Getting-Started
 
 # Getting Started with the RAPIDS Accelerator on GCP Dataproc
  [Google Cloud Dataproc](https://cloud.google.com/dataproc) is Google Cloud's fully managed Apache
- Spark and Hadoop service. The quick start guide will go through:
+ Spark and Hadoop service. Please see [Software Requirements](../download.md#software-requirements) section for complete list of Dataproc versions supported by RAPIDS plugin. The quick start guide will go through:
```

Comment:
For each line, we normally limit it to only 100 characters. Could you split it into multiple lines?